starters/remix/app/components/resolvers/Config.tsx

```diff
@@ -0,0 +1,38 @@
+import { type Config } from '@measured/puck'
+import '@measured/puck/puck.css'
+import {
+  ParagraphFaq,
+  ParagraphHero,
+  ParagraphCta,
+  ParagraphCardGroup,
+  ParagraphTestimonial,
+  ParagraphLogoGroup,
+  ParagraphViewReference,
+  ParagraphWebform,
+} from '~/components/resolvers'
+
+export const config: Config = {
+  root: {
+    // fields: {
+    //   title: fieldText,
+    //   showTitle: {
+    //     type: "select",
+    //     options: [
+    //       { label: "No", value: "false" },
+    //       { label: "Yes", value: "true" },
+    //     ],
+    //   },
+    //   summary: fieldTextArea,
+    // },
```

Comment:
If this comments are commented for a reason, we should put a comment about it

Reply:
Nice catch, since this is an Editor feature is WIP and for testing purposes

Reply:
got it removed